assignments/arrays.js

```diff
@@ -61,34 +61,61 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 //     arr[i]; // 1,2,3,4
 // }
 
-// ==== Challenge 1 ====
+// ==== Challenge 1 ==== Car 33 is a' + *car year goes here* *car make goes here* *car model goes here*`
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+// console.log("Car 33 is a " + inventory[32].car_year + ", " + inventory[32].car_make + " " + inventory[32].car_model + ".");
```

Comment:
```suggestion
// console.log( `Car 33 is a ${inventory[32].car_year} ${ inventory[32].car_make}  ${inventory[32].car_model}.` );
```
I just wanted to recommend this alternate syntax, your answer was correct. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals